CODEOWNERS

```diff
@@ -1 +1,5 @@
 *       @frankfarzan @mengqiy @Shell32-Natsu
+
+# When dependencies are updated.
+*/package.json @mengqiy @Shell32-Natsu
+*/package-lock.json @mengqiy @Shell32-Natsu
```

Comment:
I think you want to configure the reviewers settings in snyke instead of doing this through code owners. Also note that more specific rules here will take precedence over general ones.

Reply:
Per https://support.snyk.io/hc/en-us/articles/360004032117#pr-assignment, PR assignee feature is only supported for private repos. We may have to go through this route unless you have strong objection. 

> Also note that more specific rules here will take precedence over general ones.
Yes. The last matching pattern takes the most precedence.